e2e/features/step_definitions/any-ccd-page.steps.ts

```diff
@@ -20,3 +20,11 @@ Then(/^I should see "(.+)"$/, async function (text) {
 When(/^I click "(.+)"$/, async function (toClick) {
     await anyCcdPage.click(toClick);
 })
+
+Then(/^the "(.+)" tab is seen$/, async function (tabName: string) {
```

Comment:
could make this generic by having the content contains passed in as a parameter

Reply:
@ryandac Updated code